internal/devicescale/devicescale.go

```diff
@@ -46,3 +47,17 @@ func GetAt(x, y int) float64 {
 
 	return s
 }
+
+// ScreenScaleAt returns the screen scale at (x, y).
+// x and y are in device-dependent pixels and must be the top-left coordinate of a monitor, or 0,0 to request a "global scale".
```

Comment:
When (0, 0) is given, this function just returns the scale for the monitor at (0, 0), right? What is the "global scale"?

(There is the same issue at `GetAt`).

Also, from this comment, I was not sure what's the difference between `GetAt` and `ScreenScaleAt`.

Reply:
This is hard to phrase, so let me explain:

- (0, 0) is specified in practice ONLY for APIs that have no per-screen scale support (e.g. js).
- Passing (0, 0) to APIs that do have per-screen scale support return the scale of the screen at (0, 0), if any exists; if none exists, they return the scale of the primary screen.

If you can phrase this better, please suggest something.

As for GetAt vs ScreenScaleAt, I'll explain this further.

Reply:
Yeah probably this API is not good and we should define a 'dont-care' value rather than (0, 0). It's ok to leave this as it is.